Sources/xcproj/PBXProj+Helpers.swift

```diff
@@ -1,6 +1,28 @@
 import Foundation
 import PathKit
 
+// MARK: - PBXProj Extension (Public)
+
+public extension PBXProj {
+
+    /// Returns root project.
+    public var rootProject: PBXProject? {
+        return objects.projects[rootObject]
+    }
+
+    /// Returns root project's root group.
+    public var rootGroup: PBXGroup {
+        guard let rootProject = self.rootProject else {
+            fatalError("Missing root project")
```

Comment:
Is `fatalError` suitable here?
Maybe we can make `rootGroup` optional (`public var rootGroup: PBXGroup?`).

Reply:
I think it's more suitable than requiring to unwrap this property which will be nil only in specific situations when accessed before project is correctly setup. In my opinion it would be better if API would not allow to create objects in such invalid or incomplete states (what it does now as far as I understand), but not sure what it will take to change it in this direction.

Reply:
I agree with @ilyapuchka here. If we use an optional attribute, the developer could wrongly access that attribute and get unexpected behaviours as a result. If we `fatalErrror` they would know that they are doing something wrong and it'd save them some debugging time.

Reply:
Understood!👌